testsuite/test-generate.sh

```diff
@@ -1086,7 +1087,7 @@ for hbaseversion in 0.98.3-hadoop2 0.98.9-hadoop2
 do
     for hadoopversion in 2.6.0
     do
-	for zookeeperversion in 3.4.6
+	for zookeeperversion in 3.4.6 3.4.7
```

Comment:
I'd like to limit the permutation of zookeeper tests in non-zookeeper sections.  How about we leave this one as 3.4.6 (b/c its running against an older hadoop) ... and ... (see below)